lib/gtfs.ex

```diff
@@ -62,46 +33,31 @@ defmodule Gtfs do
     GenServer.call(server, :all_routes)
   end
 
-  @spec timepoints_on_route(Route.id(), GenServer.server() | nil) :: [Timepoint.id()]
-  def timepoints_on_route(route_id, server \\ nil) do
+  @spec timepoint_ids_on_route(Route.id(), GenServer.server() | nil) :: [StopTime.timepoint_id()]
+  def timepoint_ids_on_route(route_id, server \\ nil) do
     server = server || __MODULE__
-    GenServer.call(server, {:timepoints_on_route, route_id})
+    GenServer.call(server, {:timepoint_ids_on_route, route_id})
+  end
+
+  @spec stop_times_on_route(Route.id(), GenServer.server() | nil) :: [Stop.id()]
+  def stop_times_on_route(route_id, server \\ nil) do
+    server = server || __MODULE__
```

Comment:
`server \\ __MODULE___`

Reply:
Idk about this function specifically, but there's [at least one place](https://github.com/mbta/skate/blob/89fd84c96ff079e88e48faa74318ccfa6ca02ba3/lib/skate_web/controllers/health_controller.ex#L9) where one of these functions gets called with `nil` from a controller, because in tests the server gets dependency injected, and the `||` is necessary.